src/pages/ContributorPage.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from "react"
+import process from "process"
+import ContributorCard from "@/components/ContributorCard"
+import ReactPaginate from "react-paginate"
+
+function ContriButorsPage() {
+  // hooks for contributor section
+  const [contributors, setContributors] = useState([])
+  const [loading, setLoading] = useState(true)
+
+  // variable for pagination
+  const itemsPerPage = 40
+  // hooks for pagination
+  const [pageCount, setPageCount] = useState(0)
+  const [itemOffset, setItemOffset] = useState(0)
+  const [currentItems, setCurrentItems] = useState(null)
+
+  // fetch contributor data
+  const getContributorData = async () => {
+    setLoading(true) // set loading to true when the fetch request is initiated
+    await fetch(`https://api.github.com/repos/priyankarpal/ProjectsHut/contributors?per_page=100`, {
```

Comment:
add per page 300 so that new users list can show here

``` suggestion
    await fetch(`https://api.github.com/repos/priyankarpal/ProjectsHut/contributors?per_page=300`, {
```